labirynt.cpp

```diff
@@ -138,24 +172,26 @@ char getEvent()
     }// pobierz znak
 void doEvent(char c) {
     c = toupper(c);
-    unsigned char strzalka =c;//zmienna tylko do strzalek
+    string strzalka;
+    strzalka[0] =c;//zmienna tylko do strzalek
```

Comment:
co to w ogóle jest? Nie prościej było `getEvent()` dostosować do `doEvent()`, a nie odwrotnie?